src/components/Header/index.jsx

```diff
@@ -3,7 +3,7 @@ import "./header.scss";
 import HeaderImg from "../../assets/images/undraw_teacher_35j2.svg";
 
 class Header extends Component {
-    render() { 
+    render() {
```

Comment:
De acordo com a issue #5, haveria um espaço antes do `(`:
```suggestion
    render () {
```